### PR TITLE
Variable type declaration

### PR DESCRIPTION
**Pull request by iellms**
Should allow you to use builtin types to declare variables.   Let me know if it works for y'all.

### CODE REVIEW

**Review by ejcohe22 — APPROVED** [review]
Looks good Im going to pull in to hopefully get some presentation ready

File: src/proj10BittingCerratoCohenEllmer/bantam/parser/Parser.java [hunk 1]
```diff
@@ -68,7 +68,9 @@ private void advanceIfTokenMatches(Token.Kind tokenKindExpected) {
         }
         if (currentToken.kind == tokenKindExpected) {
             advance(); // move on to the next token
-        } else {
+        } else if (currentToken.kind == VAR && tokenKindExpected == IDENTIFIER){
```
Comment (KitchenTable99) [inline src/proj10BittingCerratoCohenEllmer/bantam/parser/Parser.java:71]: Can you explain a little bit more why? Beyond just saying "we are okay"

**Review by KitchenTable99 — CHANGES_REQUESTED** [review]
Looks good, but I want a little bit more detailed comment

File: src/proj10BittingCerratoCohenEllmer/bantam/parser/Parser.java (hunk 1 above)
Comment (iellms) [reply src/proj10BittingCerratoCohenEllmer/bantam/parser/Parser.java:71]: check out my updated comment

**Pull request merged**